<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabhkleditor</class>
 <widget class="QWidget" name="tabhkleditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0">
   <item>
    <layout class="QVBoxLayout" name="vL_editor_1" stretch="0,0,0">
     <item>
      <widget class="QLabel" name="l_editor_origin">
       <property name="text">
        <string>原始HKL文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="lV_editor_origin">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hL_editor_pB1">
       <item>
        <widget class="QPushButton" name="pB_editor_choose">
         <property name="text">
          <string>选择HKL文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_editor_count">
         <property name="text">
          <string>已选0个</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vL_editor_2" stretch="0,0,0,0">
     <property name="spacing">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>25</number>
     </property>
     <item>
      <layout class="QFormLayout" name="fL_editor">
       <item row="0" column="1">
        <widget class="QComboBox" name="cB_editor_method">
         <property name="currentText">
          <string>X^p</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>X^p</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>X*ln(p*X)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>X/p</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lE_editor_params">
         <property name="toolTip">
          <string>支持begin:step:end或single两种格式，用英文逗号分隔</string>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="placeholderText">
          <string>此处输入修改参数范围</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="L_editor_method">
         <property name="text">
          <string>编辑方式</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="l_editor_params">
         <property name="text">
          <string>参数范围</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="l_editor_norm">
         <property name="text">
          <string>是否归一化</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="cB_editor_norm">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="l_editor_min">
         <property name="text">
          <string>归一最小值</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="l_editor_max">
         <property name="text">
          <string>归一最大值</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="dSB_editor_min">
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="dSB_editor_max">
         <property name="minimum">
          <double>100.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100000.000000000000000</double>
         </property>
         <property name="value">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="l_editor_range">
         <property name="text">
          <string>编辑范围</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cB_editor_range">
         <item>
          <property name="text">
           <string>F2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>sigma(F2)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>both</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hL_editor_start">
       <item>
        <widget class="QPushButton" name="pB_editor_start">
         <property name="text">
          <string>开始自动编辑</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignVCenter">
        <widget class="QLabel" name="l_editor_start">
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="bar_editor">
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vL_editor_3" stretch="0,0,0">
     <item>
      <widget class="QLabel" name="l_editor_modified">
       <property name="text">
        <string>编辑后HKL文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="lV_editor_modified">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hL_editor_pB2">
       <item>
        <widget class="QPushButton" name="pB_editor_send">
         <property name="text">
          <string>发送到求解器</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_editor_new_count">
         <property name="text">
          <string>总计0个</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
